packages/react-ui/src/widgets/CategoryWidgetUI/CategoryWidgetUI.js

```diff
@@ -45,7 +46,8 @@ function CategoryWidgetUI(props) {
     animation,
     filterable,
     searchable,
-    isLoading
+    isLoading,
+    localizationMessages
```

Comment:
Why the need to pass localizationMessages as prop? I think they easiest approach here would be to read it, in the widgets, from a simple

```
import { useIntl } from 'react-intl';

function MyComponent() {
  const intl = useIntl();

  return (
    <div>
      <p>{intl.formatMessage({ id: 'greeting' })}</p>
      {/* Other components */}
    </div>
  );
}
```

Reply:
That leaves us with probably the main point here, which is lib initialization with the correct locale... Probably there are several options...

Reply:
I'm not totally sure, but maybe there is an option if we set it up upper, in the widgets (with access to the redux). The widgets could wrap up the UI, with a provider around them, and the provider could ideally get the locale from redux (just get / set locale in redux, not messages). This is pure theory, haven't tested id